app/forms/password_form.rb

```diff
@@ -29,4 +30,9 @@ def submit(params)
   def user_valid?
     @user.errors.empty?
   end
+
+  def password_valid?
+    @user.password = password
+    @user.valid?
+  end
 end
```

Comment:
Actually, I would update `FormPasswordValidator` so that we only need to update the validation in one place.